XamarinCommunityToolkit/Actions/Snackbar/iOS/SnackbarLayout.ios.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace Xamarin.CommunityToolkit.Actions.Snackbar.iOS
+{
+	class SnackbarLayout
```

Comment:
This class looks exactly the like [this one ](https://github.com/xamarin/XamarinCommunityToolkit/pull/261/files#diff-7f717804b106b22365cfc3e752af5545R5). Can you keep just one and rename this file to
 SnackbarLayout.ios.macos.cs